cpp/CMakeLists.txt

```diff
@@ -151,6 +151,12 @@ if(USE_BACKEND STREQUAL "CUDA")
         "
         )
   endif()
+elseif(USE_BACKEND STREQUAL "TENSORRT")
+  message(STATUS "-DUSE_BACKEND=TENSORRT, using TensorRT backend.")
+  set(NEURALNET_BACKEND_SOURCES
+    neuralnet/trtbackend.cpp
+    )
+  set(CUDA_USE_STATIC_CUDA_RUNTIME OFF CACHE INTERNAL "")
```

Comment:
One more question that would help in testing and merging this - do you remember why you needed to add this line? For me, it's compiling okay without this line, and having this CUDA_USE_STATIC_CUDA_RUNTIME value set in the cache makes me unable to compile the CUDA version *after* I switch my cmake from TensorRT back to CUDA. Because I think after building with TensorRT, now this value is still in force even after going back to other backends because of setting it to cache.